x-pack/plugins/security_solution/cypress/integration/alerts_detection_rules_eql.spec.ts

```diff
@@ -43,14 +56,36 @@ import {
 import {
   createAndActivateRule,
   fillAboutRuleAndContinue,
-  selectEqlRuleType,
   fillDefineEqlRuleAndContinue,
+  fillScheduleRuleAndContinue,
+  selectEqlRuleType,
 } from '../tasks/create_new_rule';
 import { esArchiverLoad, esArchiverUnload } from '../tasks/es_archiver';
 import { loginAndWaitForPageWithoutDateRange } from '../tasks/login';
 
 import { DETECTIONS_URL } from '../urls/navigation';
 
+let expectedUrls = '';
+eqlRule.referenceUrls.forEach((url) => {
+  expectedUrls = expectedUrls + url;
+});
+let expectedFalsePositives = '';
```

Comment:
If you're not expecting these values to change it would be nice use `const` with a `.map` or `.reduce` for these guys. I know these are copy/pasted from elsewhere, but I've got the `const` version in the EQL rule tests.